lib/boombox/internal_bin/elixir_endpoints/pull_sink.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Boombox.InternalBin.ElixirEndpoints.PullSink do
+  @moduledoc false
+  use Membrane.Sink
+
+  alias Boombox.InternalBin.ElixirEndpoints.Sink
+
+  def_input_pad :input,
```

Comment:
How many pads of this type can exist? Wouldn't `max_instances` be applicable here?